extensions/amp-selector/0.1/validator-amp-selector.protoascii

```diff
@@ -43,7 +44,10 @@ tags: {  # <amp-selector> for AMP (autoplay attribute allowed)
     value: ""
   }
   # <amp-bind>
-  attrs: { name: "[selected]" }
+  attrs: {
+    name: "[selected]"
+    value_regex_casei: "none|focus|select"
```

Comment:
'value_regex_casei: "none|focus|select"' needs to go to the `attrs: { name: "keyboard-select-mode" }` rule above.